components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/BUILD.gn

```diff
@@ -30,8 +30,10 @@ executable("brave_vpn_wireguard_service") {
     ":version_resources",
     "common",
     "service",
+    "status_tray",
     "//base",
     "//base:base_static",
+    "//brave/components/brave_vpn/common/wireguard",
     "//build/win:default_exe_manifest",
     "//chrome/install_static:install_static_util",
```

Comment:
how did this ever get in here before you added https://github.com/brave/brave-core/pull/18960/files#diff-89ddcd9ed3c4baf29fffec235e5062d0c5c7c4694c00b99ed985e38e4f322d6dR9 ?
This is explicitly not allowed here

Reply:
discussed in Slack and created follow up https://github.com/brave/brave-browser/issues/31306